### DISCUSSION TITLE
How to check metrics from a stored model?

### DISCUSSION THREAD

**Discussion by iranzo** [online-ml/river#687] [Q&A] [answered]
Hi,
I've moved to riverml for training my model dinamically and using 'dill' to store/restore the trained model from disk.

As I've some batches for initial training, I wanted to see the metrics once the batch has been loaded, trained with each one.

The model gets changing each time I do a metric.update, so that works...

Do I need to also save/store the CM object (cm = metrics.ConfusionMatrix() and metrics.ROCAUC() ) or those can be obtained from the model itself somehow?

Thanks

**Comment by gbolmier** [comment, marked as answer]
Hey @iranzo, you'll have to persist your metric objects, the models only store the learnt parameters. Hope it helps :)

**Reply by iranzo** [reply]
Ok, so I'll use 'dill' for storing all of them then!

Thanks!
